### DISCUSSION TITLE
In the Recent Activity panel I need to see *all* my recent activity

### DISCUSSION THREAD

**Discussion by SetTrend** [community/community#114731] [Other Feature Feedback, Questions, & Ideas]
I'm very active in issueing tickets or in participating in tickets from projects I feel they don't work properly.

Sometimes, the issues I'm participating in are getting closed for being solved or an explanation has been given. They don't appear in the **Recent Activity** panel then. So, it's very cumbersome for me to find these issues in order to evaluate the information, solution or pull request.

Please, in the **Recent Activity** panel, show _all_ my recent activity, regardless of state. And let _me_ decide the maximum number or time span of content in the **Recent Activity** panel.

Providing a simple text filter, filtering on issue text and repository name, would be very helpful then.

**Comment by Manjunathhs-0003** [comment]
Custom Notifications: Check if the platform allows for customizing notifications. You may be able to set up notifications for specific types of activity, such as when an issue you've participated in is closed or when a new comment is added.

Bookmarking or Starring: Some platforms allow users to bookmark or star issues or pull requests. This could be a way for you to keep track of specific items you're interested in, even if they're closed or resolved.

Using Labels or Tags: If the platform supports labeling or tagging issues, you could use this feature to categorize and filter your activity. For example, you could tag issues you're actively participating in or interested in following up on.

Third-Party Tools: Explore third-party tools or browser extensions that provide additional features and customization options for managing project tickets. Some of these tools might offer advanced filtering or search capabilities that could help you find and track your activity more effectively.

Providing Feedback: If the platform has a feedback or feature request mechanism, consider providing feedback about your experience and suggesting improvements, such as the ability to customize the Recent Activity panel or add text filtering options.
